packages/authentication/src/types.ts

```diff
@@ -22,3 +23,31 @@ export interface UserProfile {
   name?: string;
   email?: string;
 }
+
+export type Credentials = {
+  email: string;
```

Comment:
What if the credentials include telephone number or username?

Reply:
> What if the credentials include telephone number or username?

#2246 - to be addressed.

Reply:
@dreamdevil00 Good point. And #2246 is supposed to provide a solution for customizing the fields. (Thanks @dougal83 )